feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableGridItem.kt

```diff
@@ -123,7 +123,7 @@ fun TimetableGridItem(
 
             Row(
                 modifier = Modifier
-                    .weight(1.5f, fill = false)
+                    .weight(2f, fill = false)
```

Comment:
Thanks! Do you have any idea why this works 👀

Reply:
Previously the Row height was not enough, so I tried increasing this value

<img src="https://github.com/DroidKaigi/conference-app-2023/assets/7196624/79e5d9f3-f349-4cae-bf96-e632a5dd6e08" width=300>

Reply:
Sorry for bothering you. But Could you add some preview functions to confirm small items? Maybe we can pass modifier.size or something to confirm.

Reply:
I have no idea. I think the existing preview functions are enough.

Reply:
By the way, the following code works also. 
```
            Text(
                modifier = Modifier.weight(1f, fill = false), // notice 
                text = timetableItem.title.currentLangTitle,
                style = titleTextStyle,
                overflow = TextOverflow.Ellipsis,
            )

            Row(
                modifier = Modifier
                    .weight(1f, fill = false) // notice
                    .padding(top = TimetableGridItemSizes.titleToSchedulePadding),
            ) {
               ...
            }
```

Reply:
Hello, I did [a30a61e](https://github.com/DroidKaigi/conference-app-2023/pull/1104/commits/a30a61e2ceec4a06a9aa1e13c556c98c6a8ad3a1) @takahirom